### DISCUSSION TITLE
Environment variable for blocklist

### DISCUSSION THREAD

**Discussion by MVlaar-GH** [TechnitiumSoftware/DnsServer#432] [Ideas]
According to [the documentation](https://github.com/TechnitiumSoftware/DnsServer/blob/master/DockerEnvironmentVariables.md), you can enable blocking. To roll out completely automated it would be nice if there's also a variable for setting which blocklist we actually want to use.

**Comment by ShreyasZare** [comment]
Thanks for the request. Will get that added. It would be best that you add this as an Issue since they are kept track of.

**Comment by MVlaar-GH** [comment]
I will create an issue, thank you for the reply.
